src/queues.h

```diff
@@ -73,8 +74,8 @@ int queues_notification_insert(struct notification *n);
  *
  * @param new replacement for the old notification
  *
- * @return true, if a matching notification has been found and is replaced
- * @return false, else
+ * @return true: a matching notification has been found and is replaced
+ * @return false: else
```

Comment:
```suggestion
 * @return false: otherwise
```
It was there before this PR, but lets keep things consistent.

---

Here you're using two `@return`'s for a boolean return value but in `window_is_fullscreen`, `dbus_get_fdn_daemon_info` etc you're using two `@retvals`, I assume `@retval` would be the correct one for this as well.

Reply:
> Here you're using two @return's for a boolean [...]

Yeah, that was a mistake. Thanks for the info.

Reply:
> Edit: I'm not sure who has the @ RetVal handle but you just pinged them with this commit message! :)

I'm not sure about that. It got in my mind, too. But it was right after the push. But I thought it might be fun :grin: